extensions/dacpac/src/wizard/api/dacFxConfigPage.ts

```diff
@@ -79,19 +79,19 @@ export abstract class DacFxConfigPage extends BasePage {
 		return true;
 	}
 
-	protected async createDatabaseTextBox(): Promise<azdata.FormComponent> {
+	protected async createDatabaseTextBox(title: string): Promise<azdata.FormComponent> {
```

Comment:
nit: But this doesn't appear to have any reason to be async

Reply:
Hmm looks like this is the case for several methods in this wizard. I'll do some cleanup in a separate PR.